### ISSUE TITLE
There is no information about deck when there is no cards to learning

### ISSUE THREAD

**Issue by ps-goog** [ankidroid/Anki-Android#5027]
###### Reproduction Steps

1. Select any deck where there are card to learn
![20181007_085634](https://user-images.githubusercontent.com/43900825/46579308-f1ce9280-ca0e-11e8-9275-ebf0d9971554.jpg)

You can see information about deck
2. If there is no cards to learn there isn't this information

###### Expected Result
You should show information with a lot of nulls and number of card in deck
![screenshot_20181007-084633_ankidroid](https://user-images.githubusercontent.com/43900825/46579324-30fce380-ca0f-11e8-9247-3f28f12c75ad.jpg)

###### Actual Result
You can see only title of deck

###### Debug info
AnkiDroid Version = 2.9alpha41
Android Version = 8.0.0
ACRA UUID = dd27ecef-281f-411a-b121-c44b1ca1bdb7

###### Research
*Enter an [ x ] character to confirm the points below:*

[  ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[  ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[  ] I have searched for similar existing issues here and on the user forum

**Comment by mikehardy** [comment]
I believe we match AnkiDesktop behavior in this regard - which is one of the criteria for behavior - if Anki Desktop does something in general we will do exactly that, with exceptions for the mobile environment but I don't think that applies here? I could be wrong, but if we differ from the Anki Desktop deck picker, that's the question

**Comment by timrae** [comment]
I assume this is on a tablet? On smaller devices you'll just see a message "Daily study limit reached"

**Comment by timrae** [comment]
Actually on phones there's a "secret" way to show that information. Click on the three numbers on the right of any deck and it will open that screen

**Comment by ps-goog** [comment]
Yes, you are right "suecret" runs... and I confirm, it's impossible to show this information when learning process was ended

**Comment by mikehardy** [comment]
Yeah - I'd never seen that in phone mode - but the secret does work. Maybe we can show that screen regardless of whether there are cards due or not?

**Comment by ps-goog** [comment]
We spent more time to discuss instead of showing this data always :-)
I needed this information only once then I tried to find out how many cards it's in the deck and I couldn't.
